### ISSUE TITLE
Rev C shows CHA_2.5V connected to UB11

### ISSUE THREAD

**Issue by shabaz123** [analogdevicesinc/m1k-cad#3]
Hi!
I don't have cadence software, so can only look at PDF schematics. Which is the latest schematic in PDF? I looked in the 'old' folder, and there is a rev C schematic there. I noticed CHA_2.5V is connected to integrated circuit UA11 (as expected),  but also seems to be connected to UB11. Is that intentional?

There is a CHB_5V power signal, but that is not connected to UB11, it only goes to the connector.

I'm guessing the purpose of the CHA_2.5V connection is to allow self-calibration when it is switched in using the digipot?

What is the purpose of the CHB_5V connection? Is it just to be a general purpose connection, not actually related to channel B?

Thanks!

**Comment by itdaniher** [comment]
Hello there! 

https://github.com/analogdevicesinc/m1k-cad/blob/master/02-037760-01-d-1.pdf is the latest schematic. 

The connection from the buffered 2v5 supply to UA11 and UB11 allows the output switches to offer a 50o biasing to the 2.5V virtual ground, switchable onto both channels A and B. This concept is briefly described at https://wiki.analog.com/university/tools/m1k/hw#output_stage - if you can think of better wording, feel free to edit! The 5V supply is simply a buffered / fixed voltage rail.

In short, the connector naming reflects the electrical grouping with regards to the original amplifier - you didn't find a bug, just a hidden (for now) feature.

Regards!
Ian

PS - Allegro Viewer can be grabbed for free from http://www.cadence.com/products/pcb/pages/downloads.aspx, runs fine in a Virtual Machine.

**Comment by shabaz123** [comment]
Hi Ian,

Thank you for the information and super-quick response!
Thank you for this great project - very exciting times, exploring with the hardware and software currently!
